auth/server/src/main/java/com/seagates3/authorization/Authorizer.java

```diff
@@ -79,6 +79,10 @@ ServerResponse authorize(Requestor requestor,
     ServerResponse serverResponse = null;
     AuthorizationResponseGenerator responseGenerator =
         new AuthorizationResponseGenerator();
+    // CopyObject operation is not allowed for anonymous user
```

Comment:
can you create a class PublicAuthorizer  which will have method validatePublicAccess()   we need to keep list of Actions Prohibited in a config File

Reply:
Fixed.Please recheck